ui/src/app/subnets/views/SubnetDetails/SubnetSummary/SubnetSummary.tsx

```diff
@@ -1,7 +1,233 @@
+  const vlan = getVlanById(vlans, subnet.vlan) || null;
+  const fabric = vlan ? getFabricById(fabrics, vlan.fabric) : null;
+
+  useEffect(() => {
+    dispatch(spaceActions.fetch());
+    dispatch(vlanActions.fetch());
+    dispatch(fabricActions.fetch());
+  }, [dispatch]);
 
-const SubnetSummary = (): JSX.Element => (
-  <TitledSection title="Subnet summary" />
-);
+  return (
+    <TitledSection title="Subnet summary">
+      <Row>
+        <Col size={6}>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-name">
+                Name
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-name">{subnet.name}</p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-cidr">
+                CIDR
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-cidr">{subnet.cidr}</p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-gateway-ip">
+                Gateway IP
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-gateway-ip">{subnet.gateway_ip}</p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-dns">
+                DNS
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-dns">{subnet.dns_servers}</p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-description">
+                Description
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-description">{subnet.description}</p>
+            </Col>
+          </Row>
+        </Col>
+        <Col size={6}>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted">
+                <span id="subnet-managed-allocation">Managed allocation</span>{" "}
+                <Tooltip
+                  message="MAAS allocates IP addresses 
+from this subnet, exluding the 
+reserved and dynamic ranges."
+                  position="btm-right"
+                >
+                  <Icon name="information" />
+                </Tooltip>
+              </p>
+            </Col>
+            <Col size={4}>
+              <p aria-labelledby="subnet-managed-allocation">
+                {subnet.managed ? "Enabled" : "Disabled"}
+              </p>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-active-discovery">
+                <span id="subnet-active-discovery">Active discovery</span>{" "}
+                <Tooltip
```

Comment:
It looks like this tooltip should only appear if `subnet.managed === true` https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/app/partials/subnet-details.html#L365